packages/typescript/cli/ts-compile.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 set -o xtrace
-exec tsc --noEmit
+tsc -p ./tsconfig.json
```

Comment:
any particular reason to remove the `--noEmit` flag? It serves to just check the code, instead of outputting .js files.

Reply:
ah gotcha, ok I will add back.